change table name user to ops_user

johndeange opened the pull request:
## What changed

We found that our development Postgres DB in Azure is enforcing a stricter constraint than the Official Postgres Docker image, i.e. the Azure DB does not allow tables to be named `user` (since it is a keyword).

This PR renames the `user` table to `ops_user` 

## How to test

Automated tests pass.

## Definition of Done Checklist
- [x] OESA: Code refactored for clarity
- [x] OESA: Dependency rules followed
- [x] Automated unit tests updated and passed
- [x] Automated integration tests updated and passed
- [x] Automated quality tests updated and passed
- [x] Automated load tests updated and passed
- [x] Automated a11y tests updated and passed
- [x] Automated security tests updated and passed
- [x] 90%+ Code coverage achieved
- [x] Form validations updated

jonnalley reviewed (approved):
looks like there was also some changes to the backend in the `PortfolioCansAPI` class?

johndeange commented:
@jonnalley Yes - there was a query that was returning results in an inconsistent order (did not have an order by) that was making the tests flaky.  Also it was using the old Query API so I updated it to use the new SQLAlchemy v2 syntax and removed the flaky part of the test.
